sabnzbd/assembler.py

```diff
@@ -135,11 +135,13 @@ def run(self):
                                 sabnzbd.NzbQueue.end_job(nzo)
 
                         if unwanted_file:
-                            logging.warning(
-                                T('In "%s" unwanted extension in RAR file. Unwanted file is %s '),
-                                nzo.final_name,
-                                unwanted_file,
-                            )
+                            # Don't repeat the warning after a user override of an unwanted extension pause
+                            if nzo.unwanted_ext == 0:
```

Comment:
Shouldn't it be `2`?
https://github.com/sabnzbd/sabnzbd/blob/develop/sabnzbd/nzbstuff.py#L1383-L1384

Reply:
The only way I see for nzo.unwanted_ext to have a non-zero value at this point is after a resume, so whether one checks for `== 0` or `!= 2` shouldn't make a difference. I used the first for consistency with the check a few lines down.